bridges/primitives/polkadot-core/src/parachains.rs

```diff
@@ -71,7 +68,7 @@ impl From<u32> for ParaId {
 #[derive(
 	PartialEq, Eq, Clone, PartialOrd, Ord, Encode, Decode, RuntimeDebug, TypeInfo, Default,
 )]
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Hash, MallocSizeOf))]
```

Comment:
should we roll this back, not sure what this do 🙈 , but I saw that the crate was not maintained anymore (removed from parity-common)

Reply:
how are you using mallocsizeof if at all? it was removed a long time ago https://github.com/paritytech/substrate/issues/12658

Reply:
so this is a left over right?